### ISSUE TITLE
about the final results in the paper [3]

### ISSUE THREAD

**Issue by super-shayan** [cure-lab/LTSF-Linear#16]
Hello, 

Could you please clarify as to why the results for the DLinear are now different in the updated paper? Is it because of a different seed?

**Comment by mixiancmx** [comment]
We update the learning rate and batch size in some scripts. Now, the learning rate and batch size are consistent in one dataset, and Linear NLinear and DLinear can use the same scripts.

**Comment by super-shayan** [comment]
I am sorry I did not understand, you are saying you did it in "some" scripts but then say they are consistent in only "one" dataset? Also, I cannot see the scripts for the individual channel wise linear layer model, was that updated?

**Comment by mixiancmx** [comment]
Why results are different?
- Sorry, I mean we update the learning rate and batch size of some scripts. Then some results of DLinear in the updated paper is different from our previous paper.

What is updated?
- Now, we use same learning rate and batch size for different forecasting steps of each dataset. 

Scripts for DLinear-I?
- For the scripts for the individual channel wise linear model. I am not sure if you mean DLinear-I in our previous paper. To use DLinear-I, you can add “—individual” to the scripts and set the model to DLinear.

**Comment by super-shayan** [comment]
Thank you for your clarification. I will now try to reproduce the experiments with the same hyperparameters, and following our previous discussion, I will set the seed to "2021". Also, I did not entirely understand why the DLinear-I was dropped in the latest version of the paper. 

Also, I wanted to ask did the hyperparameters change because of more hyperparameter optimization?

**Comment by mixiancmx** [comment]
Hi, you can use our scripts to reproduce our results.

Why DLinear-I is dropped?
- Although DLinear-I can achieve best results in some cases, the purpose of our paper is questioning the effectiveness of transformers instead of proposing new models. Therefore, to save room, we only use three simple linear model which share weights across channels. If we add DLinear-I, probably we also need to add Linear-I and NLinear-I.

More hyper parameter optimization?
- No, on the contrary, we try to set more general hyper parameters for Linear models. Previously, we set different parameters for different forecasting steps, i.e 96 192 336 720, of some dataset, in order to achieve a good performance. Now, we use same parameters for all forecasting steps and sacrifice some performance. We even did not optimize the hyper parameters for Linear and NLinear. We directly apply DLinear’s hyper parameters on them.

**Comment by super-shayan** [comment]
I understand it can become difficult to include and tune more models, and why DLinear-I wasn't opted for in the new version of the paper. I also understand and agree with your point on the hyperparameter optimization. Thank you for the clarification!
